starter/script.js

```diff
@@ -1 +1,111 @@
 // put code here!
+var name = prompt("what is your name?"); //.toUpperCase();
+  if (name !=null); {
+  document.write = ("Welcome" + name +", Congratulations on your new job as a web developer intern. Your annual salary is $35,000 per year. I'm here to help you find an apartment.")
+  }
+};
+var choice = prompt("What part of the city would you like to live?NW, SW, NE, SE").toString();.toUpperCase(); {// change input to string and uppercase
+
+  if (choice == "NW"); { // display info for NW and prompt user to type answer
+  document.write("Northwest has great bars, resturants and tons of night life. \
+  You have Champange taste on a beer budget. Unfortunately you cannot afford \
+  $,2000 per month in rent on your miserable intern salary \
+    Do you get 2 roommates or uber on nights and weekends.");
+
+    var roomMateOrUber = prompt("type R for roommates or U for Uber").toUpperCase();{ // find a way to restrict input to either of the two answers only
+      if (roomMateOrUber == "R");{
+      document.write("So you miss your college days, dorm rooms, roommates, and drama?\
+       Your first roommate is great and pay his rent on time but the secnd one loses \
+       her job and can't make the rent.")
+      };
+        var KickOrAirbnb = prompt("Do you kick her out so you can find a new roommate \
+        or make her sleep on the couch and do AirBnB in her old room? \
+        Type 'K' to Kick her out or 'A' for AirBnB").toUpperCase(); { // make user choose btw Kick friend out or AirBnB
+
+          if (KickOrAirbnb == "K"); { // display the info if user chooses to kick friend out
+          document.write("Reasonable choice. Everyone needs to pay their own way, \
+           who cares if you loose your best friend from 5th grade in the process. \
+            As the saying goes, 'If you want a friend in DC get a dog!'");
+          };
+          else if (KickOrAirbnb == "A") { // display info if user chooses AirBnB
+            document.write("AirBnB is a great way to cover your missing rent and help if he \
+            couldn't pay rent he should be glad he can sleep on the couch. \
+            AirBnB works out well for two weeks until your landlord notices all the traffic \
+            in and out the apartment. You are fined, and evicted for violating your lease. \
+            You are forced to sleep in your car and shower in your office gym.");
+          };
+
+          else {
+            document.write("Please type either R or U"); // force user to make a choice by repeating KickOrAirbnb
+            return KickOrAirbnb;
```

Comment:
`return` is usually used to get a value out of a function - what is the purpose of this one here?